serving/src/main/java/ai/djl/serving/plugins/DependencyManager.java

```diff
@@ -199,6 +199,8 @@ private String getOrtVersion(String djlVersion) throws IOException {
                         + ".pom";
         try (InputStream is = new URL(maven).openStream()) {
             DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+            dbf.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
+            dbf.setXIncludeAware(false);
             DocumentBuilder db = dbf.newDocumentBuilder();
```

Comment:
recommendation is to also set the following on the factory `dbf.setExpandEntityReferences(false);`

Reply:
> recommendation is to also set the following on the factory `dbf.setExpandEntityReferences(false);`

Fixed